### ISSUE TITLE
what should happen next?

### ISSUE THREAD

**Issue by swipis** [lhark/truckersmp-cli#8]
hello I trying to run ETS2 multiplayer but cant get launcher to run. here is the terminal output:

http://imgur.com/a/H1tTG

And then nothing happens
wine version 2.0.0
ubuntu 16.04

please help!

**Comment by lhark** [comment]
Actually it looks like the launcher ran correctly according the logs you provided. The game even seems to start correctly looking at the last 4 lines of the log.

Have you tried running  the game from steam ? And steam obviously needs to be launched using wine. It won't work if you use the native linux version of steam.

Do you get an error message or does it just exit without doing anything ?

**Comment by swipis** [comment]
steam is running and exiting without doing anything.

**Comment by swipis** [comment]
EDIT: 
I run this command  and I get nothing:
_WINEPREFIX=~/wineprefix64 ./truckersmp-cli /home/swipis/wineprefix64/drive_c/Program\ Files\ \(x86\)/steam/steamapps/common/Euro\ Truck\ Simulator\ 2/_ 

but if I run without WINEPREFIX=~/wineprefix64 MP is running!

**Comment by lhark** [comment]
I did a tests, and looks like you started steam in a different WINEPREFIX than truckersmp-cli.

If I had to guess, I would say you started steam with the default WINEPREFIX (.wine) and then tried to start truckersmp-cli with WINEPREFIX=~/wineprefix64.

I reckon your fix of using .wine works because both .wine and .wineprefix64 are 64bits installations.

To confirm that that's indeed what is happening, try to start steam using WINEPREFIX=~/wineprefix64

If that doesn't work then I'm completly wrong ^^
